### PR TITLE
feat: add a proto & GORM/Gen demo

### PR DESCRIPTION
**Pull request by chenghonour**
#### What type of PR is this?
feat

#### What this PR does / why we need it (English/Chinese):
English: add a proto & GORM/Gen demo with hertz
中文：新增结合proto&GORM/Gen的hertz框架样例

### CODE REVIEW

**Comment by li-jin-gou** [comment]
ci need to solve.

**Review by li-jin-gou — CHANGES_REQUESTED** [review]
I think it would be clearer to use hertz_gorm_gen instead of hertz_gen and you would need to add it in the readme.

**Comment by chenghonour** [comment]
> I think it would be clearer to use hertz_gorm_gen instead of hertz_gen and you would need to add it in the readme.

You are right, have changed to a better name

File: bizdemo/hertz_gorm_gen/biz/handler/user/user_service.go [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+	})
+	if err != nil {
+		resp.Code = user.Code_ParamInvalid
+		resp.Msg = err.Error()
+		c.JSON(200, resp)
+		return
+	}
+
+	resp.Code = user.Code_Success
+	resp.Msg = "创建记录成功"
```
Comment (justlorain) [inline bizdemo/hertz_gorm_gen/biz/handler/user/user_service.go:57]: I think we should use English.

Comment (chenghonour) [reply bizdemo/hertz_gorm_gen/biz/handler/user/user_service.go:57]: ohh，it would be nice

**Comment by justlorain** [comment]
LGTM

**Review by li-jin-gou — CHANGES_REQUESTED** [review]
add desc to https://github.com/cloudwego/hertz-examples/blob/main/README.md and https://github.com/cloudwego/hertz-examples/blob/main/README_CN.md

File: bizdemo/hertz_gorm_gen/cmd/generate.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
```
Comment (li-jin-gou) [inline bizdemo/hertz_gorm_gen/cmd/generate.go:19]: I think we need to introduce the use of gen in this project in the readme.

**Review by li-jin-gou — CHANGES_REQUESTED** [review]

**Review by li-jin-gou — APPROVED** [review]

**Comment by li-jin-gou** [comment]
LGTM

**Pull request merged**